### PR TITLE
Feature/roc 7623 managing manual documents

### PR DESCRIPTION
**Pull request by daveoh**
### JIRA link (if applicable) ###
https://github.com/hmcts/cmc-claim-store/pull/1627

### Change description ###
Manage documents event

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

### CODE REVIEW

File: definition/data/sheets/CaseEventToComplexTypes.json [hunk 1]
```diff
@@ -31,5 +31,15 @@
...
+    {"ID": "StaffDocument","CaseEventID": "ReferToJudge","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentLink","EventElementLabel": "Link","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    {"ID": "StaffDocument","CaseEventID": "ReferToJudge","CaseFieldID": "staffUploadedDocuments","ListElementCode": "receivedDatetime","EventElementLabel": "Document received on","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "StaffDocument","CaseEventID": "ManageDocuments","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentName","EventElementLabel": "Name","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "StaffDocument","CaseEventID": "ManageDocuments","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentType","EventElementLabel": "Type","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
+    {"ID": "StaffDocument","CaseEventID": "ManageDocuments","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentTypeOther","EventElementLabel": "What type of document is it?","FieldDisplayOrder": 2,"DisplayContext": "OPTIONAL"},
```
Comment (tharacka) [inline definition/data/sheets/CaseEventToComplexTypes.json:41]: Are the FieldDisplayOrders correct on these lines?

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -252,5 +252,21 @@
...
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReferToJudge", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReferToJudge", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReferToJudge", "UserRole": "caseworker-cmc-solicitor", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReferToJudge", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ProvideDirections", "UserRole": "caseworker-cmc", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ProvideDirections", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ProvideDirections", "UserRole": "caseworker-cmc-solicitor", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ProvideDirections", "UserRole": "caseworker-cmc-judge", "CRUD": "CR"},
```
Comment (Valkrys) [inline definition/data/sheets/AuthorisationCaseEvent.json:263]: Provide Directions event is duplicated by my pr

Comment (daveoh) [reply definition/data/sheets/AuthorisationCaseEvent.json:263]: This is based from the PR for ROC-7242. This shouldn't cause any merge problems but we will need to be careful

File: definition/data/sheets/CaseEventToComplexTypes.json (hunk 1 above)
Comment (daveoh) [reply definition/data/sheets/CaseEventToComplexTypes.json:41]: Probably not. We will have to double check this once ROC-7242 is done.

**Pull request merged**